pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation_test.go

```diff
@@ -462,6 +475,77 @@ func TestExtractLibInfo(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:                "all with mutate_unlabelled off",
+			pod:                 common.FakePodWithAnnotation("admission.datadoghq.com/all-lib.version", "latest"),
+			containerRegistry:   "registry",
+			expectedPodEligible: -1,
+			expectedLibsToInject: []libInfo{ // TODO: Add new entry when a new language is supported
```

Comment:
I've noticed that the complete list appears several times in these tests, maybe we can extract it to a `var` defined somewhere in this file and reuse it.